src/main/java/tech/ypsilon/bbbot/discord/command/VerifyCommand.java

```diff
@@ -0,0 +1,151 @@
+                    "Generally speaking, the KIT Discord Server is accessible for everyone. If you are currently "
+                            + "studying at KIT, consider verifying yourself in order to access certain private "
+                            + "areas such as information channels. In order to begin the verification process, send "
+                            + "me a private message containing `kit verify uxxxx`¹. You should recieve an email "
+                            + "to your student.kit.edu account shortly containing a three digit code. Enter this code "
+                            + "like this: `kit verify 000`². After entering the correct code, you may access all "
+                            + "areas reserved for students and alumni. Alternatively if for example you currently "
+                            + "have no access to your student.kit.edu mailbox you can send us a photo of your "
+                            + "student id. Just make sure to censor the image, surname and matriculation number.\n"
+                            + "-----\n"
+                            + "¹ instead of uxxxx, enter your personal u-code\n"
+                            + "² instead of 000, enter the verification code sent by mail",
+                    false
+            )
+            .build();
+
+    private Session mailSession;
+
+    public VerifyCommand() {
+        Properties mailSessionProperties = new Properties();
+        mailSessionProperties.put("mail.smtp.auth", true);
+        mailSessionProperties.put("mail.smtp.starttls.enable", "true");
+        mailSessionProperties.put("mail.smtp.host", SettingsController.getValue("mail.smtp.host"));
+        mailSessionProperties.put("mail.smtp.port", SettingsController.getValue("mail.smtp.port"));
+        mailSessionProperties.put("mail.smtp.ssl.trust", SettingsController.getValue("mail.smtp.ssl.trust"));
+
+        mailSession = Session.getInstance(mailSessionProperties, new Authenticator() {
+            @Override
+            protected PasswordAuthentication getPasswordAuthentication() {
+                return new PasswordAuthentication(
+                        (String) SettingsController.getValue("mail.smtp.username"),
+                        (String) SettingsController.getValue("mail.smtp.password")
+                );
+            }
+        });
+    }
+
+    @Override
+    public String[] getAlias() {
+        return new String[]{"verify"};
+    }
+
+    @Override
+    public String getDescription() {
+        return "E-Mail Verifizierung um die Student-Rolle zu erhalten";
+    }
+
+    @Override
+    public void onExecute(GuildMessageReceivedEvent event, String[] args) {
+        event.getMessage().delete().queue();
+        event.getAuthor().openPrivateChannel().flatMap(privateChannel ->
+                privateChannel.sendMessage("For security reasons, the `verify` command "
+                        + "is only allowed per direct message")).queue();
+    }
```

Comment:
Why is help bilungual but this message only english? You might also consider using EmbedUtil.